docs/notes-base/platforms.md

```diff
@@ -93,6 +93,7 @@ It will run on the following versions of Android:
 * 6.0 (Marshmallow)
 * 7.x (Nougat)
 * 8.x (Oreo)
+* 9.x (Pie)
```

Comment:
This change should probably be in a separate PR, or at least a separate commit

Reply:
Sure, I'll submit a separate PR.